packages/react/index.js

```diff
@@ -1,6 +1,38 @@
 'use strict';
 
+function testMinification() {
+  if (testMinification.name === 'testMinification') {
+    // We are not minified.
+    // This might be a Node environment where DCE is not expected anyway.
+    return;
+  }
+  if (process.env.NODE_ENV === 'development') {
+    // We expect this method only to be called in production.
+    throw new Error('This is unreachable');
+  }
+  try {
+    const source = testMinification.toString();
+    if (source.indexOf('toString') === -1) {
```

Comment:
Similar thing here, this works, but is kinda misleading.

`const source = testMinification.toString();` could be replaced with `const source = String(testMinification);` and this check would still work.